script/dev-deploy

```diff
@@ -106,8 +106,11 @@ def set_config_path(custom: str, origin: str) -> str:
 def set_resource_constraints(memory_cpus: Tuple[str, str]) -> str:
     memory, cpus = memory_cpus
     return (
-        f" --memory={memory} " if memory else ""
-        f" --cpus={cpus} " if cpus else ""
+        f" --memory={memory} "
+        if memory
+        else "" f" --cpus={cpus} "
+        if cpus
+        else ""
```

Comment:
``` Python3
return "".join(
    f" --memory={memory} " if memory else "",
    f" --cpus={cpus} " if cpus else "",
)
```
use join to avoid `black` format